test/integration/routes/collaborator.spec.ts

```diff
@@ -14,35 +14,35 @@ import { authentication } from '../../../src/middleware/authentication.middlewar
 
 import {
```

Comment:
Same for the test directory, I would have created a folder for `github` routes and unit/controller